packages/esm-appointments-app/src/index.ts

```diff
@@ -26,9 +26,9 @@ import appointmentsDashboardComponent from './appointments.component';
 import homeAppointmentsComponent from './home';
 import appointmentStatusComponent from './appointments/scheduled/appointments-by-status.component';
 import earlyAppointmentsComponent from './appointments/scheduled/early-appointments.component';
-import patientAppointmentsDetailedSummaryComponent from './patient-chart/patient-appointments-detailed-summary.component';
-import patientAppointmentsOverviewComponent from './patient-chart/patient-appointments-overview.component';
-import patientUpcomingAppointmentsComponent from './patient-chart/patient-upcoming-appointments-card.component';
+import patientAppointmentsDetailedSummaryComponent from './patient-appointments/patient-appointments-detailed-summary.component';
+import patientAppointmentsOverviewComponent from './patient-appointments/patient-appointments-overview.component';
+import patientUpcomingAppointmentsComponent from './patient-appointments/patient-upcoming-appointments-card.component';
```

Comment:
I just changed this directory I had recently created to what I thing is a more apt name.

Reply:
Nit: but it'd be nice if we got rid of the [duplicated context ](https://github.com/kettanaito/naming-cheatsheet#avoid-context-duplication) in a subsequent refactor.  `Patient appointments` already establishes the patient context, so child components in that directory using the patient prefix is duplicative. There's probably a case for getting rid of the patient prefix altogether.